config/settings.py

```diff
@@ -28,7 +28,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['localhost', '127.0.0.1']
```

Comment:
Это лучше определить через переменные окружения, чтобы можно было запускать твой код не только локально
**.env**
```
ALLOWED_HOSTS=localhost,127.0.0.1
```
```suggestion
ALLOWED_HOSTS = env.list('ALLOWED_HOSTS')
```

Reply:
Также через `.env` нужно задать переменные `SECRET_KEY` и `DEBUG`. Не забудь после этого актуализировать файл `.env.example`

Reply:
у меня ошибка вызывает эта строка. Почему-то не видет локалхост если так написать.

Reply:
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.